src/sdk/wire_bytes.rs

```diff
@@ -27,41 +28,62 @@ pub fn encode<T: Serialize>(payload: &T) -> TofnResult<BytesVec> {
     })
 }
 
+/// Serialize a value using bincode and log errors
 pub fn serialize<T: ?Sized>(value: &T) -> TofnResult<BytesVec>
 where
     T: serde::Serialize,
 {
-    match bincode::serialize(value) {
-        Ok(bytes) => Ok(bytes),
-        Err(err) => {
-            error!("serialization failure: {}", err.to_string());
-            Err(TofnFatal)
-        }
-    }
+    // Create serialization options for bincode.
+    // The default options don't bound pre-allocation size,
+    // use little-endian and varint encoding, and reject trailing bytes.
+    let options = DefaultOptions::new()
```

Comment:
@sdaveas We should update `tofnd` to also use these options instead of the default `bincode::serialize`. Maybe `tofn` can expose our `encode`/`decode` function for `tofnd` to use?

Reply:
tofn should definitely expose something for tofnd.  No sense maintaining a separate copy of these settings inside tofnd, too.